src/services/blocks/BlocksService.ts

```diff
@@ -327,46 +334,70 @@ export class BlocksService extends AbstractService {
 		parentHash: Hash,
 		block: Block
 	) {
-		let parentParentHash: Hash;
-		if (block.header.number.toNumber() > 1) {
-			parentParentHash = (await api.rpc.chain.getHeader(parentHash))
-				.parentHash;
+		const perByte = api?.consts?.transactionPayment?.transactionByteFee;
+		const extrinsicBaseWeight = api?.consts?.system?.extrinsicBaseWeight;
```

Comment:
dq: Are these per-chain configurable? I.e. can other chains have their own values here?

Reply:
Yup

https://github.com/paritytech/polkadot/blob/f70e52c056688bc3f1480701cf2a1f5827c0e6a0/runtime/polkadot/src/lib.rs#L244

And ExtrinisicBaseWeight is from benchmarking but you can see it can be configured:
https://github.com/paritytech/polkadot/blob/f70e52c056688bc3f1480701cf2a1f5827c0e6a0/runtime/common/src/lib.rs#L151